examples/material-ui/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "private": true,
+  "scripts": {
+    "start": "rescripts start",
+    "build": "rescripts build",
+    "test": "rescripts test",
+    "eject": "rescripts eject"
+  },
+  "dependencies": {
+    "@material-ui/core": "^4.4.3",
+    "eslint-config-prettier": "^6.3.0",
+    "material-ui": "^0.20.2",
```

Comment:
Is this package "material-ui" in use?  Perhaps this was installed by mistake the first time you tried to install Material UI.